client/state/action-types.js

```diff
@@ -194,7 +194,12 @@ export const KEYRING_SERVICES_REQUEST_SUCCESS = 'KEYRING_SERVICES_REQUEST_SUCCES
 export const LAYOUT_FOCUS_SET = 'LAYOUT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_SET = 'LAYOUT_NEXT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_ACTIVATE = 'LAYOUT_NEXT_FOCUS_ACTIVATE';
+export const MEDIA_ITEMS_RECEIVE = 'MEDIA_ITEMS_RECEIVE';
```

Comment:
The problem in your link is that the action creator could dispatch with a single or multiple items. Here, we only have a single action type that will always include an array of items.